NEW_README.md

```diff
@@ -0,0 +1,110 @@
+# Engineering Process Project Requirements Document
+
+- Owner: @just-mitch
+- Approvers:
+  - @joeandrews
+  - @charlielye
+  - @iAmMichaelConnor
+  - @aminsammara
+  - @0xrafi
+  - @rahul-kothari
+  - @LHerskind
+  - @LeilaWang
+  - @dbanks12
+  - @PhilWindle
+  - @nventuro
+  - @Rumata888
+  - @critesjosh
+- Target PRD Approval Date: 2025-03-14
+
+## Key words
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+## Introduction
+
+- "Complex work" by engineering MUST have a Design Document (DD).
+- Determining whether work is "complex" is the responsibility of engineering team leads and product managers. Good candidates include:
+  - Introduces new concepts, data structures, algorithms, services, or contracts
+  - Interface or protocol changes
+  - Affects multiple products or users (including internal and external)
+  - Major refactoring or restructuring of code
+- A Design Document MUST have an associated Project Requirements Document (PRD).
+- PRDs and DDs MUST be kept up to date via pull requests.
+- Changes to the requirements in a PRD require re-approval by the individuals or groups specified in the original PRD.
+- Changes to the design which affect its compliance with the PRD MUST be re-approved by the individuals or groups specified in the original PRD.
+
+## Project Requirements Document (PRD)
```

Comment:
Maybe add a little bullet point explaining the main objective of the PRD vs the DD: "to explain the what, and not the how".